src/components/AuthenticationProvider.tsx

```diff
@@ -33,21 +33,21 @@ export const AuthenticationProvider: ChildrenFC = ({ children }) => {
     }
   }, []);
 
-  const login = useCallback((newToken: string, newUserName: string) => {
-    localStorage.setItem(TOKEN_KEY, newToken);
-    setUserName(newUserName);
-  }, []);
+  const login = useCallback(
+    (newToken: string, newUserName: string, newUserEmail: string) => {
+      localStorage.setItem(TOKEN_KEY, newToken);
+      setUser({ name: newUserName, email: newUserEmail });
```

Comment:
This is not what I meant. Sorry for not being clearer :D. Just pass the whole user from trpc here -> 

![CleanShot 2024-01-03 at 10 48 22](https://github.com/KamilaSemotanova/tracker/assets/12488648/cfe8eb98-2b8e-4d9d-b555-dde47b65e79c)

Leave out the password.